Task 2/Task 2.1/Task 2.1/firure/Rectangle.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+
+namespace Task_2._1
+{
+    public class Rectangle : Figure, IAeraing
+
+    {
+        private double height;
+        private double width;
```

Comment:
There's no properties exposing these! How others will know about rectangle size? (it should be a part of public interface)